fvupdater.cpp

```diff
@@ -96,13 +97,26 @@ void FvUpdater::installTranslator()
 
 void FvUpdater::showUpdaterWindowUpdatedWithCurrentUpdateProposal()
 {
-    // Destroy window if already exists
-    hideUpdaterWindow();
+    qDebug() << "FvUpdater::showUpdaterWindowUpdatedWithCurrentUpdateProposal()";
+
+    if(m_mode == NORMAL) {
+        // Destroy window if already exists
+        hideUpdaterWindow();
 
-    // Create a new window
-    m_updaterWindow = new FvUpdateWindow();
-    m_updaterWindow->UpdateWindowWithCurrentProposedUpdate();
-    m_updaterWindow->show();
+        // Create a new window
+        m_updaterWindow = new FvUpdateWindow();
+        m_updaterWindow->UpdateWindowWithCurrentProposedUpdate();
+        m_updaterWindow->show();
+    } else {
+        //Get the proposed update and emit the signal
+        FvAvailableUpdate* proposedUpdate = FvUpdater::sharedUpdater()->GetProposedUpdate();
+        if (! proposedUpdate) {
```

Comment:
format - no space for `if (!proposedUpdate)`. But better condition is `if (proposedUpdate == nullptr)`